src/locale/zh-cn.js

```diff
@@ -57,9 +57,21 @@ export default moment.defineLocale('zh-cn', {
     calendar : {
         sameDay : '[今天]LT',
         nextDay : '[明天]LT',
-        nextWeek : '[下]ddddLT',
+        nextWeek : function (now) {
+            if (now.week() !== this.week()) {
+                return '[下]dddLT';
+            } else {
+                return '[本]dddLT';
```

Comment:
Is it necessary to explicitly mention `本` for the current week?

```suggestion
                return 'dddLT';
```

Reply:
It is about accurate, when your are in Saturday, you say Sunday, it will be the next week's Sunday, means '下'(next) week, not '本'(this) week
But this nearest rule is not explicated, so you probably don't know which week it is. while on the other hand, if you say Monday of this week (本周一）, Sunday of this week（本周日）, it's explicated located an accurate day which refer to current day, share the same week, in certain culture (with the Sunday belong to previous week or next week rule setup).

And before the commit mentioned in my PR which resulted wrong calendar output, the code did this too, which is the proper way to make the representation of the day as explicated accurate as possible and simple. Just as you said Monday of next week (下周一），the Monday of this week（本周一) is simply same rule and same clear. while just "Monday" would be unclear if you are in a day which is more close to the next Monday.